.semaphore/cp_dockerfile_build.yml

```diff
@@ -117,19 +117,19 @@ blocks:
             - . publish-test-results
             - artifact push workflow target/test-results
             - artifact push workflow target --destination target-AMD
-  - name: Deploy AMD confluentinc/cp-kafkacat
+  - name: Deploy AMD confluentinc/cp-kcat
     dependencies: ["Build, Test, & Scan AMD"]
     run:
       when: "branch = 'master' or branch =~ '^[0-9]+\\.[0-9]+\\.x$' or branch =~ '^[0-9]+\\.[0-9]+\\.[0-9]+(-cp[0-9]+)?-rc[0-9]+$'"
     task:
       jobs:
-        - name: Deploy AMD confluentinc/cp-kafkacat ubi8
+        - name: Deploy AMD confluentinc/cp-kcat ubi8
```

Comment:
We need to keep the name as kafkacat for 7.0.x and 7.1.x. Only from 7.2.x we're distributing it as kcat